Plugins/Flow.Launcher.Plugin.Program/Main.cs

```diff
@@ -442,12 +448,26 @@ public List<Result> LoadContextMenus(Result selectedResult)
                     Title = Context.API.GetTranslation("flowlauncher_plugin_program_disable_program"),
                     Action = c =>
                     {
-                        _ = DisableProgramAsync(program);
-                        Context.API.ShowMsg(
-                            Context.API.GetTranslation("flowlauncher_plugin_program_disable_dlgtitle_success"),
-                            Context.API.GetTranslation(
-                                "flowlauncher_plugin_program_disable_dlgtitle_success_message"));
-                        Context.API.ReQuery();
+                        _ = Task.Run(async () =>
+                        {
+                            try
+                            {
+                                var disabled = await DisableProgramAsync(program);
+                                if (disabled)
+                                {
+                                    ResetCache();
```

Comment:
How does the ResetCache call work when we are resetting multiple times ? For example user disables one program after another.